### DISCUSSION TITLE
Templates - Support for end-user documentation

### DISCUSSION THREAD

**Discussion by michaelbrewer** [coder/coder#12591] [RFCs]
When creating a template there is a single readme, and is often geared towards to "template admin" user or the operator of the compute resources that a template needs. 

I would like to propose that there be documentation geared toward the users who want to create a workspace. 

These docs should be easy to find (currently the description field is too short and does not support hyper links or formatting), and should support markdown to allow template designers to include:

- capabilities of the workspace
- usage docs
- feedback links
- support links 
- contribution guides 
- etc. 

This will help developers know what this workspace can do for them, how to use it and how they could expand on the template. 

Also there is going to be a large learning curve for how to migrate workloads off their local machines. And any guides to do so could be at the template level.

**Comment by deansheather** [comment]
The best way to probably do this is to make the README be the end user document, and add another markdown file like `template.md` in the template directory. People who are working on the template can refer to that document in their IDE when necessary, and you can mention it in the README for discoverability (although linking to it won't work as far as I'm aware).

Honestly, we should probably do this for the example templates to set a good example.

**Reply by michaelbrewer** [reply]
Makes sense. Good examples would help people copy and change :)
